Copilot/README.md

```diff
@@ -113,7 +111,7 @@ curl -o $package_path $url && [ "$(sha256sum $package_path | awk '{ print $1 }')
 
 In Windows:
 ```powershell
-$version = "1.0.2"
+$version = "1.0.3"
 $url = "https://download.microsoft.com/download/4d8b9fec-9ae3-44c5-bcc9-f329ee96cbc1/microsoft-sharepointembedded-copilotchat-react-1.0.3.tgz"
 $expected_checksum = "B9ABEBF07A597BA6FA5A2838ECBE6F1D53BD5EBD8A0FC5FBC2D21764BFF9C11C"
```

Comment:
If you will proceed with changing the version, make sure these checksum values are still correct.

Reply:
```bash
$ curl https://download.microsoft.com/download/4d8b9fec-9ae3-44c5-b
cc9-f329ee96cbc1/microsoft-sharepointembedded-copilotchat-react-1.0.3.tgz | sha256sum
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 32031  100 32031    0     0   124k      0 --:--:-- --:--:-- --:--:--  125k
b9abebf07a597ba6fa5a2838ecbe6f1d53bd5ebd8a0fc5fbc2d21764bff9c11c  -
```